libs/blocks/table/table.js

```diff
@@ -375,6 +375,15 @@ function handleSection(sectionParams) {
       sectionRowTitle.setAttribute('scope', 'row');
     }
   }
+
+  rowCols.forEach((col) => {
+    const formattingTags = col.querySelector('strong, em, del, code, sub, sup');
```

Comment:
Could it be that the `col` has a `strong` inside it, but it's already nested deeper in the DOM? Maybe something like  `.querySelector(':scope > :is(strong, em, del, code, sub, sup)');` would address this issue. And that could eliminate the need to have the `formattingTags.parentNode.tagName === 'DIV'` check as well